New callout partial refund stage (#1347)
vtexdocs/dev-portal-content: docs/guides/Integration-Guides/payments-integration-guide/split-payouts-on-payment-provider-protocol.md

## Cart Scenarios

When an order is placed, one of three scenarios are present in the cart:

1. Products only from the Marketplace.
2. Products from the Marketplace, and one or more sellers.
3. Products only from seller(s).

### Marketplace Products only

As the order contains products only from the Marketplace, there is no split of payment, calculation of commissioning, or split payload to be sent. The Marketplace will receive the full order payment value.

### Marketplace and Sellers Products only

When an order contains products from the Marketplace and sellers (or only from sellers), a split payout is required.
In this scenario, the VTEX gateway is responsible for:

* Calculate commissioning per seller of each order.
* Send split payload data to the payment processor.
* Process the complete or partial refund in split operations, if required.

> ⚠️ The split payload is sent only if the payment provider supports this scenario.

> A supposed situation is shown below to explain how a **calculation breakdown** for split is performed:

Reviewer: ```suggestion
An illustrative example is shown below to explain how a **calculation breakdown** for split is performed:
```

]
```

4. The following containers will be sent with these values:

>    a. 92.36 for the Marketplace.
>    b. 73.18 for sellerX.
>    c. 34.08 for sellerY.

Reviewer: ```suggestion
   a. 92.36 for the marketplace.
   b. 73.18 for seller X.
   c. 34.08 for seller Y.
```

### Authorization Stage

> For transactions where the split in _authorization_ is enabled, we send the _recipients_ in the authorization payload. The _“recipient’s”_ object array contains data on the values to be passed on to each seller and the Marketplace involved in the order.

Reviewer: ```suggestion
For transactions where the split in _authorization_ is enabled, we send the _recipients_ in the authorization payload. The _“recipient’s”_ object array contains data on the values to be passed on to each seller and the marketplace involved in the order.
```

2. The cart is made up of products from sellerA.
3. An order with the value of 45.00 is registered.
4. The following containers will be sent with these values:

>    a. 7.2 for the Marketplace.
>    b. 37.8 for sellerA.

Reviewer: ```suggestion
   a. 7.2 for the marketplace.
   b. 37.8 for seller A.
```

3. The cart was made up of items from sellerA.
4. The value to be refunded from sellerA item is 20.00.
5. The following containers will be sent with these values:

>    a. 3.2 for the Marketplace.
>    b. 16.8 for sellerA.

Reviewer: ```suggestion
   a. 3.2 for the marketplace.
   b. 16.8 for seller A.
```

}'
```

### Cancellation Stage

> > ℹ️ In this version of the Split Payouts, there is no change in the normal Cancellation payload.

Reviewer: ```suggestion
> ℹ️ In this version of the Split Payouts, there are no changes in the normal cancellation payload.
```

}'
```

> For more information about payment splits on VTEX, see [Split Payment](https://help.vtex.com/en/tutorial/split-de-pagamento--6k5JidhYRUxileNolY2VLx?&utm_source=autocomplete#).

Reviewer: ```suggestion
For more information about payment splits on VTEX, see [Split Payment](https://help.vtex.com/en/tutorial/split-payment--6k5JidhYRUxileNolY2VLx).
```
